### PR TITLE
MICROVERSE PROJECT: Embedding Images and Video

### PR DESCRIPTION
**Pull request by anapdh**
Created features in this branch:

• A navbar on the top of the screen;
• An iframe video from YouTube that displays;
• Description and avatar of the video's owner account;
• A sidebar with recommended videos.
• HTML & CSS3;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+![screenshot](./assets/SCREENSHOT.png)
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
```
Comment (teekaytech) [inline README.md]: Your live demo link is not working. Kindly replace it when you're done making all the other changes. [This link](https://rawcdn.githack.com/) might help you.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  #search-bar {
+    width: 700px;
+  }
+
+  #search-button {
+    width:10%;
+  }
+
+#img-logo {
+  height: 60%;
```
Comment (teekaytech) [inline styles.css]: Be consistent with your spaces and tabs. Kindly fix this section and other sections of your CSS rules that are not well indented

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+#create-img, #apps-img, #notifications-img {
+  width: 50px;
+  height: 60%;
+}
+
+section {
+  width: 100;
+}
```
Comment (teekaytech) [inline styles.css]: You should always specify the unit when declaring CSS rules that deal with measurement. e.g. `width: 100%` or `width: 100px`, etc.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+li {
+  display: inline-flex;
+  align-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  padding: 4px;
+  font-size: 14px;
+}
```
Comment (teekaytech) [inline styles.css]: You have some repeated code in these blocks. Always try to keep your code DRY. Please try to refactor these blocks.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+/*.
+  split {
+  height: 100%;
+  top: 35px;
+  position:absolute;
+  overflow-x: hidden;
+  padding-top: 40px;
+}
+*/
```
Comment (teekaytech) [inline styles.css]: Always delete commented code / block of codes. Please remove this block.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+.split-left {
+  padding-left: 10px;
+  margin-top: 50px;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  /*
+  margin: 15px;
+  width: 68%;
+  left: 0;*/
```
Comment (teekaytech) [inline styles.css]: Delete this too.

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+  margin-top: 50px;
+  position: absolute;
+  left: 70%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  float: right;
+  /*
+  width: 30%;
+  right: 0;*/
```
Comment (teekaytech) [inline styles.css]: Please delete this too.

File: youtube-homepage.html [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+  <article class="description">
+    <img class="avatar-img" src="/assets/todp-img.png" alt="avatar">
+    <input id="subscribe-btn" type="image" src="/assets/subscribe.png" alt="subscribe">
+    <br>
+    <p>Hey ninjas, in this Git & GitHub tutorial I want to show you 
+      how you can fork a repository to your own GitHub account. You'd 
+      typically do this if you wanted to contribute to some kind of 
+      open source project.<p>
+      <br>
+      <p>---------------------------------------------------------------------------------------------</p>
```
Comment (teekaytech) [inline youtube-homepage.html]: This is not a good practice. You can use an horizontal line `<hr>` for this. You just style it up with your CSS to look like this. Besides, I'm not sure this is in the project.

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

### Hello team :wave: ;

**You've started the project well. However, a lot of changes are required before this project can be approved. Please review the points below carefully and make all the changes as required.**

**[REQUIRED CHANGES]**
_These changes must be made before the project can be approved._

- [ ] Your project structure and layout does not look like the original page structure.

- Here's the original youtube video player page, from [this url](http://archive.vn/Bss88)

o

- Here's your own youtube player page: 

y

You are expected to produce a clone of the original page. Please study the original page carefully and make your layout/design to conform with it.

- [ ] Your pull request description does not summarize the changes you're trying to merge. 
- Your pull request description should be a summary of the changes you're willing to merge to master.

o

- [ ] Most of the images you referenced in your project (index.html) are not rendering. Please take some time to read and understand the requirements of this project, then make changes where necessary.

- [ ] You did not follow CSS best practices.
- I observed that you use IDs a lot as your CSS selector. You should get used to using class names as CSS selectors.
- Please take a step back and review [this lesson](https://speckyboy.com/good-bad-css-practices) on CSS good and bad practices. 

o

- [ ] Kindly check the highlighted points I made in your code and fix them.

**Please make these changes and request for another review by submitting a new code review request form when you are done.**

I'm @taofeekolalere on slack, in case you need further clarification.
Happy coding :man_technologist:

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,107 @@
+html {
+  background-color: #F9F9F9;
+  padding: 1294 x 1298;
```
Comment (ghost) [inline styles.css]: The value of the padding property is not declared properly on line 3.  Check out this[ link](https://developer.mozilla.org/en-US/docs/Web/CSS/padding) on how to write and use the padding property.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**Good work on implementing the requested changes. However, there are a few more changes needed:**

# Mandatory

1).  **You are not allowed to use screenshots/image for the image side text on the sidebar(check the circled area on the image below). Please use HTML tags to add the side texts and ensure the styling and alignment is similar to the original page.**

- [ ] Ensure that students do not use screenshots of original page as shortcut to bypass requirements. 

youtube_clone_sideimage

2). **The section below the  video does not look like the original.**

original  |  clone
---------- | ---------
bottom_original  |  bottom_clone

- Notice the subscribe button is displaying on the left and shows number of subscribers next to it.

- likes, dislikes and share icons display below the user avatar and subscribe button.

- The total number of views display on the right.

3). **The side text for the user avatar should not be a screenshot/image(check circled area on the image below). Use HTML to add the text and ensure the styling and alignment is similar to the original.**

avatar1

4). **Add a Getting started section, on your read me file, with details on how to set up your project locally, and the necessary tools required.  Check the [readme](https://github.com/microverseinc/readme-template) template for more info on this.**

**There is one more comment below.**

# Optional

- **Fix the warnings from HTML checker.**

### Goodluck😃!

**PS: You can reach me on slack(Rose Wanjohi)**

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

- [ ] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/94326247-35d04e80-ffbc-11ea-8567-9bf75c152b26.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/) ([screenshot](https://user-images.githubusercontent.com/29689273/94326203-fa358480-ffbb-11ea-81ec-b2704fb93212.png) from Pathwright)

- **Hint:** By closing `div` and `article` tags the issue will be fixed
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: styles.css (hunk 9 above)
Comment (ghost) [reply styles.css]: > The value of the padding property is not declared properly on line 3. Check out this[ link](https://developer.mozilla.org/en-US/docs/Web/CSS/padding) on how to write and use the padding property.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**There is one change you haven't implemented as requested previously. Please check the comment below.**

**While at it, you can also fix the issue page link and mit link on your read me file.**

#### Happy coding guys!

**PS: You can reach me on slack(Rose Wanjohi).**

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hi, @anapdh and @shahnaeem 
Congratulations on the work done so far, you guys are amazing. I see you have implemented most of the changes requested by the TSEs 👏 👏  👏 .

Let us quickly add some finishing touches, then we will be done.

- [ ] You still have one HTML validation errors. Use [validator.w3.org ](https://validator.w3.org/)

- [ ] Please refer to the last fix requested by the previous TSE. 
Screen Shot 2020-09-26 at 11 59 00 PM

That is it guys, make these changes and you should be good to go.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

You can merge to master.
Good luck with your next project.

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**